nova_vm/src/engine/bytecode/vm.rs

```diff
@@ -581,6 +584,26 @@ impl Vm {
                 );
                 vm.stack.push(object.into())
             }
+            Instruction::CopyDataPropertiesIntoObject => {
+                // 7.3.25 CopyDataProperties ( target, source, excludedItems )
+                // 1. If source is either undefined or null, return unused.
+                // 2. Let from be ! ToObject(source).
+                let Ok(from) = Object::try_from(vm.result.unwrap()) else {
+                    return Ok(ContinuationKind::Normal);
```

Comment:
**question:** Shouldn't we still clear the reference stack (though I suppose we don't _actually_ ever enter here with a non-object on the stack)?

Reply:
I changed this to instead panic if the result value isn't an object.